docs/manual/developer_guide.adoc

```diff
@@ -655,6 +655,29 @@ For any of the `[red]#Required#` directories that may not yet add content,
 add a `.gitkeep` file for any empty directories.
 ====
 
+=== Profiles
+
+Profiles define the set of rules and variables aligned to a compliance standard.
+
+Structurally, a profile is a YAML file that represents a dictionary.
+A profile YAML file has one implied attribute:
+
+* `id`: The primary identifier for the profile, to be referenced during evaluations. This is inferred from the file name.
+
+A profile YAML file can, optionally, include metadata about its maintainers and the implemented policy:
+
+* `metadata`: Dictionary for profile metadata.
+  * `policy_hub`: URL pointing to page or organization that publishes the policy.
+  * `current_version`: Version of the policy implemented by the profile.
+  * `maintainers`: List of people maintaining, responsible or experienced in the profile.
```

Comment:
I would mention explicitly that we expect those to be Github usernames. A slightly inconvenient issue with usernames is that they can change, but I wouldn't make a big deal out of it.

Reply:
Mentioned, but didn't make it exclusive to GH handles.